lapis-e2e/test/nucleotideInsertions.spec.ts

```diff
@@ -51,6 +60,17 @@ describe('The /nucleotideInsertions endpoint', () => {
     expect(descendingOrderedResult.data[3]).to.have.property('insertion', 'ins_22204:CAGAA');
   });
 
+  it('advancedQuery can filter out unwanted insertions', async () => {
+    const urlParams = new URLSearchParams({
+      advancedQuery: "NOT ins_5959:TAT AND country='Switzerland'",
+    });
+
+    const result = await getNucleotideInsertions(urlParams);
+    expect(result.status).equals(200);
+    const resultJson = await result.json();
+    expect(resultJson.data).to.have.length(3);
```

Comment:
What does this test actually how? How is the result related to the query? The length of this result is more or less arbitrary and doesn't really say anything about whether the insertion was filtered out or now, does it?

Naively, I would have expected that we would need to check 
1. only filter for Switzerland, check that we have a certain number of insertions for `ins_5959:TAT`
2. then include the insertion in the query and check that we don't see that insertion in the result anymore.

Reply:
For details and aggregated results, the length of the response is quite meaningful. For insertion results, it's not really meaningful. The other tests in this file also check other properties inside the array of the result.